### ISSUE TITLE
Hi, Can we support more language?

### ISSUE THREAD

**Issue by Sun-FreePort** [Grokmoo/sulis#210]
I want player game at my home language (chinese for me), but not has i18 module?

If we support translate, has some work need? is hard?

**Comment by Grokmoo** [comment]
Unfortunately this is quite a lot of work.  We would first need to factor out all of the strings we are displaying into some i18n system - this would include many places not just in the rust code but in scripts and dialog as well.  Then, the actual translation work could start.

**Comment by Sun-FreePort** [comment]
If I want to do that (i18 module), I can submit PR? (I don't join any open source project, so, lacking experience, sorry).

Init i18 functional, and step to step do that is gradual plan, so first, we only need translate a part english word and can test it, is better? <- only I think.

**Comment by Sun-FreePort** [comment]
@Grokmoo Hi, thank you and your reply.

If I want to do that (i18 module), I can submit PR? (I don't join any open source project, so, lacking experience, sorry).

Init i18 functional, and step to step do that is gradual plan, so first, we only need translate a part english word and can test it, is better? <- only I think.

**Comment by Grokmoo** [comment]
Yes, I'm happy to review any PRs.  However, I should stress that this is definitely a large undertaking.

The first step would be to setup an i18n module that can load strings from a file.  This would need to be done in a way that integrates with sulis's asset loading - probably from another asset folder type, i.e. `translations` as a new `YamlResourceKind`.  This would enable the translations to work with mods / overrides in the same manner as the other assets.

**Comment by Sun-FreePort** [comment]
> "integrates with sulis's asset loading" / "translations to work with mods / overrides"

And assert import or export to common struct -- like json / xml, them can use to Transifex or other.

Understand, I will read relational code in sulis's project, and reply my question or program design.

Good nice day for you!
